packages/at_backupkey_flutter/lib/widgets/backup_key_widget.dart

```diff
@@ -335,21 +335,25 @@ PLEASE SECURELY SAVE YOUR KEYS. WE DO NOT HAVE ACCESS TO THEM AND CANNOT CREATE
           },
         );
       } else if (Platform.isIOS) {
-        var size = MediaQuery.of(context).size;
-        await Share.shareXFiles(
-          [XFile(tempFilePath)],
-          sharePositionOrigin: Rect.fromLTWH(0, 0, size.width, size.height / 2),
-        ).then((ShareResult shareResult) {
-          if (shareResult.status == ShareResultStatus.success) {
-            ScaffoldMessenger.of(context).showSnackBar(
-                const SnackBar(content: Text('File saved successfully')));
-          }
-        });
+        if (context.mounted) {
+          var size = MediaQuery.of(context).size;
+          await Share.shareXFiles(
+            [XFile(tempFilePath)],
+            sharePositionOrigin:
+                Rect.fromLTWH(0, 0, size.width, size.height / 2),
+          ).then((ShareResult shareResult) {
+            if (shareResult.status == ShareResultStatus.success) {
+              ScaffoldMessenger.of(context).showSnackBar(
+                  const SnackBar(content: Text('File saved successfully')));
+            }
+          });
+        }
       } else {
-        final path =
-            await getSavePath(suggestedName: '$atsign${Strings.backupKeyName}');
+        final path = await getSaveLocation(
+            suggestedName: '$atsign${Strings.backupKeyName}');
+        if (path == null) return;
         final file = XFile(tempFilePath);
-        await file.saveTo(path ?? '');
+        await file.saveTo(path.path);
         if (context.mounted) {
```

Comment:
getSavePath was removed in 1.0.0 of path_selector, so I replaced it with the getSaveLocation